prime-router/src/main/kotlin/azure/ActionHistory.kt

```diff
@@ -191,7 +192,13 @@ class ActionHistory {
     fun trackActionResult(actionResult: String) {
         val tmp = if (action.actionResult.isNullOrBlank()) actionResult else "${action.actionResult}, $actionResult"
         val max = ACTION.ACTION_RESULT.dataType.length()
-        action.actionResult = tmp.chunked(size = max)[0]
+        // max is 0 for the CLOB type. we're using CLOB for the action_result now because we want
+        // bigly strings, not just small sad 2048 strings
+        action.actionResult = if (ACTION.ACTION_RESULT.dataType == SQLDataType.CLOB && max == 0) {
+            tmp
+        } else {
+            tmp.chunked(size = max)[0]
+        }
```

Comment:
I upsized the `ACTION.ACTION_RESULT` column to be `TEXT` so it accepts a lot more data. You're welcome.

Reply:
I wonder if this field still needs to hold as much given that we're no longer writing the response over PII concerns (see https://github.com/CDCgov/prime-reportstream/pull/3074)